### PR TITLE
Login

### PR DESCRIPTION
**Pull request by gharamorth**

### CODE REVIEW

File: FSBeheer/FSBeheer/FSContext.cs [hunk 1]
```diff
@@ -8,6 +8,8 @@ namespace FSBeheer
 
     public partial class FSContext : DbContext
     {
+        internal object temp;
```
Comment (Evert-Arends) [inline FSBeheer/FSBeheer/FSContext.cs]: Delete?

File: FSBeheer/FSBeheer/ViewModel/AccountVM.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using GalaSoft.MvvmLight;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FSBeheer.ViewModel
+{
+    public class AccountVM : ViewModelBase
```
Comment (Evert-Arends) [inline FSBeheer/FSBeheer/ViewModel/AccountVM.cs]: Do not

File: FSBeheer/FSBeheer/ViewModel/CreateQuestionViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        public ObservableCollection<QuestionTypeVM> QuestionTypes { get; set; }
+        public RelayCommand AddQuestionCommand { get; set; }
+
+        public CreateQuestionViewModel(QuestionnaireVM selectedQuestionnaireVM)
+        {
+            _selectedQuestionnaireVM = selectedQuestionnaireVM;
+
+            _question = new Question();
+
+            using (var context = new CustomFSContext())
```
Comment (Evert-Arends) [inline FSBeheer/FSBeheer/ViewModel/CreateQuestionViewModel.cs:55]: Please use the correct classes for this functionality as stated in the CustomFXContext

File: FSBeheer/FSBeheer/ViewModel/CreateQuestionViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+                return _selectedQuestionType;
+            }
+            set
+            {
+                _selectedQuestionType = value;
+                base.RaisePropertyChanged("SelectedQuestionType");
+            }
+        }
+
+        public void AddQuestion()
```
Comment (Evert-Arends) [inline FSBeheer/FSBeheer/ViewModel/CreateQuestionViewModel.cs:69]: What?

File: FSBeheer/FSBeheer/ViewModel/LoginViewModel.cs [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+            Message = "";
+            Account = new AccountVM();
+        }
+
+        public void VerifyLogin()
+        {
+            string username;
+            string password;
+            string salt;
+            int i = 5;
```
Comment (Evert-Arends) [inline FSBeheer/FSBeheer/ViewModel/LoginViewModel.cs]: Please remove

File: FSBeheer/FSBeheer/ViewModel/CreateQuestionViewModel.cs (hunk 4 above)
Comment (gharamorth) [reply FSBeheer/FSBeheer/ViewModel/CreateQuestionViewModel.cs:69]: What

File: FSBeheer/FSBeheer/VM/QuestionnaireVM.cs [hunk 6]
```diff
@@ -1,119 +1,70 @@
...
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.Windows.Input;
+using System;
 
-namespace FSBeheer.VM
+namespace FSBeheer.ViewModel
```
Comment (Evert-Arends) [inline FSBeheer/FSBeheer/VM/QuestionnaireVM.cs]: Wrong namespace? This should be in the VM folder? Please explain a bit further?

**Review by Evert-Arends — CHANGES_REQUESTED** [review]

File: FSBeheer/FSBeheer/VM/QuestionnaireVM.cs (hunk 6 above)
Comment (gharamorth) [reply FSBeheer/FSBeheer/VM/QuestionnaireVM.cs]: Wrong namespace yeah, it is in the VM folder

**Comment by Evert-Arends** [comment]
Even development pullen en pushen, dan merge ik m

File: FSBeheer/FSBeheer/ViewModel/QuestionnairesViewModel.cs [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+{
+    public class QuestionnairesViewModel : ViewModelBase
+    {
+        //PLURAL FORM
+        public ObservableCollection<QuestionnaireVM> Questionnaires { get; set; }
+        private QuestionnaireVM _selectedQuestionnaire;
+
+        private QuestionnaireEditView _questionnaireEditView;
+
+        public ICommand ViewQuestionnaireCommand { get; set; }
```
Comment (Evert-Arends) [inline FSBeheer/FSBeheer/ViewModel/QuestionnairesViewModel.cs]: ICommand should not be used, use RelayCommand instad.

**Review by Evert-Arends — CHANGES_REQUESTED** [review]

File: FSBeheer/FSBeheer/FSContext.cs (hunk 1 above)
Comment (Evert-Arends) [reply FSBeheer/FSBeheer/FSContext.cs]: @gharamorth Is dit opgelost?

**Review by Evert-Arends — CHANGES_REQUESTED** [review]

**Review by Evert-Arends — APPROVED** [review]

**Pull request merged**